leptonai/cli/deployment.py

```diff
@@ -38,10 +38,167 @@
     ResourceRequirement,
     ScaleDown,
     ContainerPort,
+    AutoscalerTargetThroughput,
 )
 from ..api.v1.types.photon import PhotonDeploymentTemplate
 
 
+def deprecation_warning(ctx, param, value):
```

Comment:
nit: could you change this to "autoscale_flag_deprecation_warning" so we know what it is used for?